### ISSUE TITLE
How to use my own dataset

### ISSUE THREAD

**Issue by hanshuai119** [znxlwm/pytorch-generative-model-collections#18]

**Comment by ArghyaPal** [comment]
Hi,

open the dataloader.py file
and, add

elif dataset == 'AWA':
        data_loader = DataLoader(
             datasets.ImageFolder('data/AWA', transform=transform),
             batch_size=batch_size, shuffle=True)

then, open main.py
and modify this line:
parser.add_argument('--dataset', type=str, default='mnist', choices=[ 'mnist', 'fashion-mnist', 'cifar10', 'cifar100', 'svhn', 'stl10', 'lsun-bed'],

to, parser.add_argument('--dataset', type=str, default='mnist', choices=['AWA', 'mnist', 'fashion-mnist', 'cifar10', 'cifar100', 'svhn', 'stl10', 'lsun-bed'],

**Comment by HamzahNizami** [comment]
Do the images have to be in a specific format or size.
Thanks in advance

**Comment by HamzahNizami** [comment]
> Hi,
> 
> open the dataloader.py file
> and, add
> 
> elif dataset == 'AWA':
> data_loader = DataLoader(
> datasets.ImageFolder('data/AWA', transform=transform),
> batch_size=batch_size, shuffle=True)
> 
> then, open main.py
> and modify this line:
> parser.add_argument('--dataset', type=str, default='mnist', choices=[ 'mnist', 'fashion-mnist', 'cifar10', 'cifar100', 'svhn', 'stl10', 'lsun-bed'],
> 
> to, parser.add_argument('--dataset', type=str, default='mnist', choices=['AWA', 'mnist', 'fashion-mnist', 'cifar10', 'cifar100', 'svhn', 'stl10', 'lsun-bed'],


Do the images have to be in a specific format or size.
Thanks in advance

**Comment by ArghyaPal** [comment]
No, I don't think so. Please go ahead and use your dataset.. please reply back, if you encounter any problem related to image size

**Comment by HamzahNizami** [comment]
Okay I will try soon. 

I have a dataset of my own JPG images. These images are of strawberries. The image I want to generate is of strawberries. 

My images are in colour, will this affect training on this GitHub. Do I need to change to gray-scale. 

Finally @ArghyaPal have you successfully trained this model on your dataset? Any help would be greatly appreciated. 

Thanks in advance!

**Comment by HamzahNizami** [comment]
> No, I don't think so. Please go ahead and use your dataset.. please reply back, if you encounter any problem related to image size

@ArghyaPal 

Hi, I'm currently getting the following error.

![image](https://user-images.githubusercontent.com/32271819/54887141-672c0380-4e87-11e9-83fd-6f82645241a5.png)

Even though my images are jpg as shown in the image below. Any help would be appreicated. 

![image](https://user-images.githubusercontent.com/32271819/54887132-55e2f700-4e87-11e9-83b4-766461bd86b7.png)

**Comment by HamzahNizami** [comment]
I managed to fix this issue by placing them in another folder called 1, so my directory is now, data/strawberriesBlack/1/B(1).jpg

However, my results are completely blank after 50 epochs on GAN

![image](https://user-images.githubusercontent.com/32271819/54888142-5df46400-4e92-11e9-8894-ce6c802c1d81.png)

My dataset is currently coloured, do you think I should change to grayscale? 
How big should said dataset actually be?

Thanks in advance

**Comment by evdelph** [comment]
@ArghyaPal I am having issues with either size or color scaling with these models, . I am trying to run color images of ships through GAN but I get shape errors (see issue 25). Is there a fix to this?
